sdk/security/keyvault/azcertificates/example_test.go

```diff
@@ -94,3 +94,46 @@ func ExampleClient_DeleteCertificate() {
 	// In a soft-delete enabled vault, deleted resources can be recovered until they're purged (permanently deleted).
 	fmt.Printf("Certificate will be purged at %v", *resp.ScheduledPurgeDate)
 }
+
+func ExampleClient_ImportCertificate_pfx() {
+	// This example uses `ImportCertificate` to import a PFX certificate.
```

Comment:
FYI, if you put intro text in a comment on the `func`, it renders above the example code [like this](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.7.0-beta.1#example-AuthenticationRequiredError)
```suggestion
// This example uses `ImportCertificate` to import a PFX certificate.
func ExampleClient_ImportCertificate_pfx() {
```